userdetails-cognito/src/main/groovy/au/org/ala/userdetails/CognitoUserService.groovy

```diff
@@ -858,4 +867,78 @@ class CognitoUserService implements IUserService<UserRecord, UserPropertyRecord,
         resultStreamer.complete()
     }
 
+    @Override
+    Map generateApikey(String usagePlanId) {
```

Comment:
I'm thinking this may still be useful for non Cognito deployments, can we put a feature flag on this and extract this into its own service?